qark/plugins/Dynamically_Loading_Code.py

```diff
@@ -5,25 +5,58 @@
 from modules import common
 from lib.pubsub import pub
 import lib.plyj.parser as plyj
+import lib.plyj.model as m
 
 
 class DynamicallyLoadingCodePlugin(IPlugin):
     DEX_CLASS_LOADER = r'DexClassLoader'
     CLASS_LOADER = r'loadClass'
     DYNAMIC_BROADCAST_RECEIVER = r'registerReceiver'
+    global list_BR
+    list_BR = []
 
     def __init__(self):
         self.name = 'Dynamically loading code'
 
+    # recursive function to find the loadclass function name by traversing down the AST
+    def recursive_classloader_function(self, t, filename, res):
+        if type(t) is m.MethodDeclaration:
+            if str(t.name) == self.CLASS_LOADER:
+                PluginUtil.reportInfo(fileName, self.DexClassLoaderIssueDetails(fileName), res)
+                return
+            
+        elif type(t) is list:
+            for x in t:
+                self.recursive_classloader_function(x, filename, res)
+        elif hasattr(t, '_fields'):
+            for f in t._fields:
```

Comment:
Change `f` to something more descriptive. I think `f` is supposed to stand for `field` here?